coreTable/OwidTable.test.ts

```diff
@@ -507,6 +507,95 @@ describe("time domain", () => {
     })
 })
 
+describe("linear interpolation", () => {
+    const table = new OwidTable(
+        `gdp,year,entityName,entityId,entityCode
```

Comment:
nit: I think you can drop entityId and entityCode now from the test case. Those should no longer be required to create an OwidTable